docs/pages/machine-id/manifesto.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Machine ID Manifesto
+description: A manifesto for Machine Identity
+---
+
+As time goes on, machines and services are doing more and more impactful and
+privileged work. Services are increasingly communicating with each other across
+network and company boundaries. AI agents are getting increasingly common and
+are becoming more autonomous and powerful at an accelerating pace.
+
+On the other side, services and machines are increasingly a target of attacks,
+with hackers trying to get access to critical systems, stealing API keys and
+exploiting compromises to get access to critical data and control over
+mission-critical systems.
+
+These challenges create an urgent need for trusted computing that rests on three
+pillars.
+
+### Strong identity-based authentication
+
```

Comment:
Are these shared secrets liable to be exfiltrated _because_ they do not provide a distinct identity for each service?

Reply:
Hm - I worded this poorly, I'll restructure it a little.

Reply:
80f34d1a73